tests/TestCase/Cache/Engine/ApcuEngineTest.php

```diff
@@ -113,15 +113,15 @@ public function testReadAndWriteCache()
 
         $result = Cache::read('test', 'apcu');
         $expecting = '';
-        $this->assertEquals($expecting, $result);
+        $this->assertSame($expecting, $result);
```

Comment:
```
$expecting = '';
$this->assertSame($expecting, $result);
```
needs to be replaced with
```
$this->assertNull($result);
```
since cache miss returns `null` by default.